### PR TITLE
minor

### PR DESCRIPTION
**Pull request by Ghost-017**

### CODE REVIEW

File: 2-ui/5-loading/02-script-async-defer/article.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 In modern websites, scripts are often "heavier" than HTML: their download size is larger, and processing time is also longer.
 
-When the browser loads HTML and comes across a `<script>...</script>` tag, it can't continue building DOM. It must execute the script right now. The same happens for external scripts `<script src="..."></script>`: the browser must wait until the script downloads, execute it, and only after process the rest of the page.
+When the browser loads HTML and comes across a `<script>...</script>` tag, it can't continue building the DOM. It must execute the script right now. The same happens for external scripts `<script src="..."></script>`: the browser must wait until the script downloads, execute it, and only after process the rest of the page.
 
 That leads to two important issues:
 
-1. Scripts can't see DOM elements below them, so can't add handlers etc.
+1. Scripts can't see DOM elements below them, so it can't add handlers etc.
```
Comment (paroche) [inline 2-ui/5-loading/02-script-async-defer/article.md]: This isn't right. "Scripts" is plural, so can't use "it". Either use no explicit subject, as was, or use "they".

File: 2-ui/5-loading/02-script-async-defer/article.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ There are some workarounds to that. For instance, we can put a script at the bot
 
 But this solution is far from perfect. For example, the browser notices the script (and can start downloading it) only after it downloaded the full HTML document. For long HTML documents, that may be a noticeable delay.
 
-Such things are invisible for people using very fast connections, but many people in the world still have slower internet speeds and use far-from-perfect mobile internet.
+Such things are invisible for people using very fast connections, but many people in the world still have slow internet speeds and use far-from-perfect mobile internet.
```
Comment (paroche) [inline 2-ui/5-loading/02-script-async-defer/article.md]: I think this is OK either way. More important to me would be adding changing the last phrase to " use **a** far-from-perfect mobile internet **connection**". Unless you want "internet" to be synonymous with "internet connection".

File: 2-ui/5-loading/02-script-async-defer/article.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ The following example demonstrates that:
 ```
 
 1. The page content shows up immediately.
-2. `DOMContentLoaded` waits for the deferred script. It only triggers when the script `(2)` is downloaded is executed.
+2. `DOMContentLoaded` waits for the deferred script. It only triggers when the script `(2)` is downloaded and is executed.
```
Comment (paroche) [inline 2-ui/5-loading/02-script-async-defer/article.md]: I would keep the new "and" but also take out "is": "is downloaded and executed".

File: 2-ui/5-loading/02-script-async-defer/article.md [hunk 4]
```diff
@@ -192,7 +192,7 @@ Please note that if you're using `defer`, then the page is visible *before* the
 
 So the user may read the page, but some graphical components are probably not ready yet.
 
-There should be "loading" indication in proper places, not-working buttons disabled, to clearly show the user what's ready and what's not.
+There should be a "loading" indication in proper places, set not-working buttons to disabled, to clearly show the user what's ready and what's not.
```
Comment (paroche) [inline 2-ui/5-loading/02-script-async-defer/article.md]: If it's "places" then it should be ' ..should be "loading" indication**s** '. (And if it's singular ("place"), then ' a "loading" indication in **a** (or **the**) proper place '

**Comment by iliakan** [comment]
Merged and updated manually&

**Pull request merged**